docs/tools/developer-tools/laboratory.mdx

```diff
@@ -0,0 +1,12 @@
+---
+title: Laboratory
+description: Explore a suite of developer tools for Stellar, including Laboratory
```

Comment:
Is this description accurate? This file has only Lab here.

Reply:
Updated the lab name and description